### PR TITLE
remove margin and bullets from ULs in forms

### PR DESCRIPTION
**Pull request by deadlight**
Done:
Removed margin and bullets from ULs in forms.

QA:
gulp build and there is a list in a form at the bottom of the form elements section in the demo.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -385,12 +385,34 @@
...
+                                    <input placeholder="I'm an input text" id="input" type="text">
+                                </li>
+                                <li>
+                                    <label for="input">Label</label>
+                                    <input placeholder="I'm an input text" id="input" type="text">
+                                </li>
+                                <li>
+                                    <label for="input">Label</label>
+                                    <input placeholder="I'm an input text" id="input" type="text">
+                                </li>
```
Comment (anthonydillon) [inline demo/index.html]: Missing ending ul tag

**Comment by anthonydillon** [comment]
CSS is OK but there is an issue in the markup.

**Comment by deadlight** [comment]
Oh yes. That was part-done. I had in ended to clean that up some more

**Comment by deadlight** [comment]
Fixed the HTML issue.

**Pull request merged**